src/io/GeantSetupOptionsIO.json.cc

```diff
@@ -0,0 +1,89 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2022 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file GeantSetupOptionsIO.json.cc
+//---------------------------------------------------------------------------//
+#include "GeantSetupOptionsIO.json.hh"
+
+#include <unordered_map>
+
+#include "base/Assert.hh"
+#include "base/Range.hh"
+
+namespace celeritas
+{
+namespace
+{
+//---------------------------------------------------------------------------//
+// HELPER FUNCTIONS
+//---------------------------------------------------------------------------//
+/*!
+ * Get a string corresponding to a surface type.
+ */
+const char* to_cstring(GeantSetupPhysicsList value)
+{
+    CELER_EXPECT(value != GeantSetupPhysicsList::size_);
+
+    static const char* const strings[] = {
+        "em_basic",
+        "em_standard",
+        "ftfp_bert",
+    };
+    static_assert(
+        static_cast<int>(GeantSetupPhysicsList::size_) * sizeof(const char*)
+            == sizeof(strings),
+        "Enum strings are incorrect");
+
+    return strings[static_cast<int>(value)];
+}
+
+//---------------------------------------------------------------------------//
+// TODO: template and rewrite these as a helper class
```

Comment:
I assume this is going to be tackled at another time.

Reply:
Yep! It works for the current need but when we want to add enum class I/O somewhere else, this is the "TODO" for where it'll be generalized :)